src/main/java/echo/Run.java

```diff
@@ -0,0 +1,7 @@
+//package echo;
```

Comment:
Is this file needed?

Reply:
I was trying to create a file that ran the whole project I've changed it to `main.java`